website/static/js/addProjectPlugin.js

```diff
@@ -0,0 +1,224 @@
+                                })
+                            ]),
+                            m('.f-w-lg.text-bigger','Category'),
+                            m('.category-radio.p-h-md', [
+                                ctrl.categoryList.map(function(cat){
+                                    return m('.radio', m('label', [  m('input', {
+                                        type: 'radio',
+                                        name: 'projectCategory',
+                                        value: cat.value,
+                                        checked: ctrl.newProjectCategory() === cat.value,
+                                        onchange : m.withAttr('value', ctrl.newProjectCategory)
+                                    }), cat.display_name || m('i.text-muted', '(Empty category)') ]));
+
+                                })
+                            ])
+                        ] : ''
+                    ])
+                ]),
+                m('.modal-footer', [
+                    m('button[type="button"].btn.btn-default[data-dismiss="modal"]', { onclick : ctrl.reset},  'Cancel'),
+                    ctrl.isValid() ? m('button[type="button"].btn.btn-success', { onclick : ctrl.add },'Add') : m('button[type="button"].btn.btn-success[disabled]','Add')
+                ])
+            ]),
+            processing : m('.modal-content',
+                m('.modal-content',
+                    m('.modal-header', [
+                        m('button.close[data-dismiss="modal"][aria-label="Close"]',{ onclick : ctrl.reset}, [
+                            m('span[aria-hidden="true"]','×'),
+                        ]),
+                    ]),
+                    m('.modal-body.text-left', [
+                            m('.add-project-processing', 'Saving your project...')
+                        ]
+                    )
+                )
+            ),
+            success : m('.modal-content', [
+                m('.modal-content',
+                    m('.modal-body.text-left', [
+                            m('button.close[data-dismiss="modal"][aria-label="Close"]',{ onclick : ctrl.reset}, [
+                                m('span[aria-hidden="true"]','×'),
+                            ]),
+                            m('h4.p-md.add-project-success.text-success', 'Project created successfully!')
+                        ]
+                    ),
+                    m('.modal-footer', [
+                        m('button[type="button"].btn.btn-default[data-dismiss="modal"]', {
+                            onclick : function() {
+                                ctrl.reset();
+                                ctrl.options.stayCallback.call(ctrl); // results are at ctrl.saveResult
+                            }
+                        },  'Keep Working Here'),
+                        m('a.btn.btn-success', { href : ctrl.goToProjectLink() },'Go to New Project')
```

Comment:
Should be sentence-case according to osf-style, unless there is good reason to make this title-case.